src/panels/lovelace/cards/hui-media-control-card.ts

```diff
@@ -218,25 +314,42 @@ export class HuiMediaControlCard extends LitElement implements LovelaceCard {
     const newImage = this.hass.states[this._config.entity]?.attributes
       .entity_picture;
 
-    if (!newImage || newImage === oldImage) {
+    if (!newImage) {
       this._image = newImage;
+      this.foregroundColor = undefined;
+      this.backgroundColor = undefined;
       return;
     }
 
-    if (newImage.substr(0, 1) !== "/") {
+    if (newImage === oldImage) {
       this._image = newImage;
       return;
     }
 
     fetchMediaPlayerThumbnailWithCache(this.hass, this._config.entity)
       .then(({ content_type, content }) => {
         this._image = `data:${content_type};base64,${content}`;
+        this._setColors();
       })
       .catch(() => {
         this._image = undefined;
+        this.foregroundColor = undefined;
+        this.backgroundColor = undefined;
       });
   }
 
+  private _attachObserver(): void {
+    if (typeof ResizeObserver !== "function") {
+      install();
```

Comment:
Can we do the import async here ? It's not that bad if we start observing a little later on Safari?

Reply:
```js
import("resize-observer").then((modules) => {
  modules.install();
  this._attachObserver();
});
```

Something along the lines of this? Or should I place the Module in a file and import the file. Not sure if that's better for Webpack

Reply:
this is fine

Reply:
this is fine but we should only do it if it's not a function.

Reply:
Yep. this is in the if statement above

Reply:
Well then make sure you also add a return